core/tst/software/aws/toolkits/core/credentials/sso/SsoAccessTokenProviderTest.kt

```diff
@@ -215,7 +215,7 @@ class SsoAccessTokenProviderTest {
         setupCacheStub(expirationClientRegistration)
 
         ssoOidcClient.stub {
-            stubStartDeviceAuthorization(interval = 1)
```

Comment:
This doesn't do anything since the interval time is overriden by the SlowDownException which spec says to be 5s